Source/WebInspectorUI/UserInterface/Views/DetailsSection.css

```diff
@@ -164,40 +164,36 @@ body[dir=rtl] .details-section > .header::before {
     margin-inline-start: 3px;
 }
 
-.details-section > .content {
-    display: table;
-    width: 100%;
-    border-spacing: 0;
-    border-collapse: collapse;
-}
-
 .details-section.collapsed > .content {
     display: none;
 }
 
 .details-section > .content > .group {
-    display: table-row-group;
+    display: grid;
+    grid-template-columns: [label] minmax(85px, max-content) [value] 1fr;
     border-bottom: 1px solid var(--border-color);
 }
 
-.details-section > .content > .group:has(.row.font-variation) {
-    display: unset; /* Unsets `display: table-row-group` because it affects sizing of child elements that are grid containers. */
+.details-section > .content > .group > * {
+    grid-column: 1 / -1;
 }
 
 .details-section > .content > .group:last-child {
     border-bottom: none;
 }
 
+
 .details-section > .content > .group:nth-child(even) {
     background-color: var(--background-color-intermediate);
 }
 
-.details-section > .content > .group > .row.simple {
-    display: table-row;
+.details-section > .content > .group:has(> .row.simple:last-of-type) {
+    padding-block-end: 3px;
```

Comment:
Do we really not want the padding on the end of the group when the last row isn't a `.row.simple`? I know this is how it was done previously, but now seems like the opportunity to ask why before we continue doing this...

Reply:
This seems intentional. A `.simple` row is just that, a row lacking any decoration, nested within a `.group`

Other things nested in a `.group` have more elaborate styling, like contrasting backgrounds. 

For example, in the Node details sidebar, the single `.row` holding the list of properties. Or multiple `.event-listener-section` containers for event handlers. A `padding-block-end` to the group here would break the design which expects the sections to sit flush one against the another.

<img width="831" alt="Screenshot 2023-03-08 at 13 45 30" src="https://user-images.githubusercontent.com/105809798/223858541-df787765-345e-450e-a18b-43ffd743ba5e.png">